jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -439,47 +439,6 @@ public IQ handleColibriConferenceIQ(ColibriConferenceIQ conferenceIq)
         return conference.getShim().handleColibriConferenceIQ(conferenceIq);
     }
 
-    /**
```

Comment:
I think it'd be better to cherry-pick this removal to a separate PR, it makes the overall PR diff a bit confusing otherwise.